src/components/select/select.js

```diff
@@ -1582,4 +1582,14 @@ function SelectProvider($$interimElementProvider) {
     }
     return isScrollable;
   }
+
+}
+
+function shouldHandleKey(ev) {
+  var char = String.fromCharCode(ev.keyCode);
+  var isNonUsefulKey = (ev.keyCode <= 31);
+  var isLeftRightSelectKey = (ev.keyCode >= 91 && ev.keyCode <= 93);
```

Comment:
Because we're checking for 91, 92 & 93. If we did that, we'd have:

```js
(ev.keyCode == 91 || ev.keyCode == 92 || ev.keyCode == 93)
```

which seemed even more verbose.